React/Views/RCTView.m

```diff
@@ -435,12 +435,33 @@ - (BOOL)isAccessibilitySelectorAllowed:(SEL)selector {
     if (_onAccessibilityAction != nil && accessibilityActionsNameMap[@"decrement"]) {
       isAllowed = YES;
     }
+#if TARGET_OS_OSX // TODO(macOS ISS#2323203)
+  } else if (selector == @selector(accessibilityPerformShowMenu)) {
+    if (_onAccessibilityAction != nil && accessibilityActionsNameMap[@"showMenu"]) {
+      isAllowed = YES;
+    }
+#endif
```

Comment:
Can you add a closing comment here like below just to keep the scope crystal clear if eventually multiple if defs get added within/around this? `] // TODO(macOS ISS#2323203)`

Reply:
sure